server.js

```diff
@@ -1,41 +1,37 @@
-'use strict';
-
-var express = require('express');
-var app = express();
-var logger = require('morgan');
-var debug = require('debug')('SIR');
-var error = require('debug')('SIR:error');
-
-var hbs = require('express-handlebars');
-var passport = require('passport');
-var session = require('express-session');
-
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-var multer = require('multer');
-var validate = require('./lib/validate');
-var rateLimit = require('./lib/rate-limit');
-
-var _ = require('lodash');
-var dotty = require('dotty');
-var fs = require('fs');
-var mv = require('mv');
-var path = require('path');
-var yaml = require('js-yaml');
-var async = require('async');
-var changeCase = require('change-case');
-
-var env = process.env;
-var gaToken = env.GA_TOKEN;
-var slackUrl = env.SLACK_WEBHOOK_URL;
-var channel = env.SLACK_CHANNEL;
-var botName = env.SLACK_BOT_NAME || 'SIR';
-var nodeEnv = env.NODE;
-
-function exitWithError(err) {
-  error(err);
-  process.exit(1);
-}
+import express from 'express';
+const app = express();
+import logger from 'morgan';
+import bug from 'debug'
+const debug = bug('SIR');
+const error = bug('SIR:error');
+
+import hbs from 'express-handlebars';
+import passport from 'passport';
+import session from 'express-session';
+
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+import multer from 'multer';
+import validate from './lib/validate';
+import rateLimit from './lib/rate-limit';
+
+import _ from 'lodash';
```

Comment:
Please double-check imports. I don't think Lodash is used here, possibly others, too.